src/oss/langchain/multi-agent.mdx

```diff
@@ -2,90 +2,89 @@
 
-Multi-agent systems are useful when:
+## Why multi-agent?
 
-* A single agent has too many tools and makes poor decisions about which to use.
-* Context or memory grows too large for one agent to track effectively.
-* Tasks require **specialization** (e.g., a planner, researcher, math expert).
-
-## Multi-agent patterns
-
-| Pattern                           | How it works                                                                                                                                                     | Control flow                                               | Example use case                                 |
-|-----------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------|--------------------------------------------------|
-| [**Tool Calling**](#tool-calling) | A **supervisor** agent calls other agents as *tools*. The “tool” agents don’t talk to the user directly — they just run their task and return results.                  | Centralized: all routing passes through the calling agent. | Task orchestration, structured workflows.        |
-| [**Handoffs**](#handoffs)         | The current agent decides to **transfer control** to another agent. The active agent changes, and the user may continue interacting directly with the new agent. | Decentralized: agents can change who is active.            | Multi-domain conversations, specialist takeover. |
-
-<Card
-    title="Tutorial: Build a supervisor agent"
-    icon="sitemap"
-    href="/oss/langchain/supervisor"
-    arrow cta="Learn more"
->
-    Learn how to build a personal assistant using the supervisor pattern, where a central supervisor agent coordinates specialized worker agents.
-    This tutorial demonstrates:
-    * Creating specialized sub-agents for different domains (calendar and email)
-    * Wrapping sub-agents as tools for centralized orchestration
-    * Adding human-in-the-loop review for sensitive actions
-</Card>
+When developers say they need "multi-agent," they're usually looking for one or more of these capabilities:
 
+* **Context management**: Provide specialized knowledge without overwhelming the model's context window. If context were infinite and latency zero, you could dump all knowledge into a single prompt — but since it's not, you need patterns to selectively surface relevant information.
+* **Distributed development**: Allow different teams to develop and maintain capabilities independently, composing them into a larger system with clear boundaries and independent testing.
+* **Parallelization**: Spawn specialized workers for subtasks and execute them concurrently.
 
-## Choosing a pattern
+## When to use
```

Comment:
Maybe we can kill `#when to use` and add the sequential constraints to the multiagent above?

and maybe incorporate this part if we feel it's not obvious?: ``` A single agent has too many [tools](/oss/langchain/tools) and makes poor decisions about which to use.
* [Context](/oss/concepts/context) or [memory](/oss/langchain/short-term-memory) grows too large for one agent to track effectively.```